### DISCUSSION TITLE
multinlang

### DISCUSSION THREAD

**Discussion by youssef1129** [typesnippet/antd-phone-input#97] [Q&A]
hi guys is there any way to do multilang ?

**Comment by ArtyomVancyan** [comment]
Hi @youssef1129, if you mean support for internationalization (i18n), it is defined in the #79 issue and will be available in the near future.

**Reply by youssef1129** [reply]
thank you mate

**Reply by ArtyomVancyan** [reply]
Hey @youssef1129, localization support has been added in the [latest release](https://github.com/typesnippet/antd-phone-input/releases/tag/v0.3.9) of the package. You can now use it by following the [documentation](https://github.com/typesnippet/antd-phone-input?tab=readme-ov-file#localization), and please feel free to ask any questions when having problems with integration.

**Reply by youssef1129** [reply]
thanks Artyom
